matrix/rotate_matrix.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+
+"""
+	In this problem, we want to rotate the matrix elements by 90, 180, 270 (counterclockwise)
+	Discussion in stackoverflow: 
+   https://stackoverflow.com/questions/42519/how-do-you-rotate-a-two-dimensional-array
+"""
+
+
+def rotate_90(matrix: [[]]):
+    """
+    rotate_90([[1, 2, 3, 4], [5, 6, 7, 8], [9, 10, 11, 12], [13, 14, 15, 16]])
```

Comment:
Please add ">>> " to the beginning of this line to make a doctest and then run __python3 -m doctest -v matrix/rotate_matrix.py__ to make sure that the doctest runs and passes